src/sentry_plugins/hipchat_ac/templates/sentry_hipchat_ac/configure.html

```diff
@@ -1,5 +1,38 @@
 {% extends "sentry_hipchat_ac/configure_layout.html" %}
 {% block body %}
+
+<script>
+// open sentry login in another window and then receive a message of whether user is logged in. If user is logged in, refresh the page, so that user can continue through configuration flow
+  function extractOrigin() {
+    var pathArray = location.href.split( '/' );
+    var protocol = pathArray[0];
+    var host = pathArray[2];
+    var origin = protocol + '//' + host;
+    return origin;
+  }
+
+  var popup;
+
+  function openLogin() {
+    var top = screen.height / 2 - 350
+    var left = screen.width / 2 - 300
+    popup = window.open(
+      (extractOrigin() + "/auth/login?next=" + encodeURIComponent('/auth/close/')),
+      "sentryLogin",
+      "modal, resizable, scrollbars, height=700, width=600, top="+top+", left="+left
+    );
+
+    window.addEventListener('message', function (evt) {
+      if (event.origin !== extractOrigin()) {
+        console.log("wrong domain");
```

Comment:
maybe throw a hard error here, so if Sentry is bound itll automatically handle it.

e.g.

```
throw new Error('message event received from invalid origin: ' + event.origin);
```